src/main/java/com/gargoylesoftware/htmlunit/util/NameValuePair.java

```diff
@@ -90,4 +90,8 @@ public int hashCode() {
     public String toString() {
         return name_ + "=" + value_;
     }
+
+    public NameValuePair normalized() {
```

Comment:
can you please add the jdoc for this method including the INTERNAL API hint

Reply:
done. Squashed and force pushed to PR source branch